packages/core/app-layout/docs/page-header.md

```diff
@@ -0,0 +1,67 @@
+# AppPageHeader.vue
+
+A Kong UI dynamic page header component.
+
+- [Features](#features)
+- [Requirements](#requirements)
+- [Usage](#usage)
+  - [Install](#install)
+  - [Props](#props)
+  - [Slots](#slots)
+
+## Features
+
+- Reactive updates based on `prop` value changes :rocket:
+- Slottable areas for displaying custom content, icons, etc.
```

Comment:
suggestion: Can you add an item here about the breadcrumbs (and that they are optional)?